modular_skyrat/master_files/code/modules/clothing/under/akula_jobs.dm

```diff
@@ -0,0 +1,302 @@
+		even sometimes worn underneath orbital suits such as MODs. \n\n\
+		Shoredresses apply active thermal channels and motion-powered micropumps to allow for water of the wearer's temperature choice to circulate; \
+		ensuring ample flow over not only the gills, but the rest of the wearer's skin, at the cost of occasional movement being needed when the wearer is still. \n\
+		These suits are known to come with luminescent panels that take on a bright glow when underwater, meant for signalling as well as higher visibility in deep waters. \
+		The system is meant to only be able to process water, fresh or otherwise; but unofficially, \
+		a great many chemicals or even drinks have been loaded in by adventurous or careless explorers of the New Principalities-- at fantastic personal risk to their gills. \n\n\ "
+	icon_state = "default"
+	base_icon_state = "default"
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/akula.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/akula.dmi'
+	armor_type = /datum/armor/wetsuit_under
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	can_adjust = FALSE
+	female_sprite_flags = NO_FEMALE_UNIFORM
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON
+	/// If an akula tail accessory is present, we can overlay an additional icon
+	var/tail_overlay
+
+
+/obj/item/clothing/under/akula_wetsuit/Initialize(mapload)
+	. = ..()
+	update_appearance()
+
+/obj/item/clothing/under/akula_wetsuit/Destroy()
+	. = ..()
+	var/mob/user = loc
+	if(!istype(user))
+		return
+	if(tail_overlay)
+		user.cut_overlay(tail_overlay)
+		tail_overlay = null
+	remove_wetsuit_status_effect(user)
+
+/obj/item/clothing/under/akula_wetsuit/equipped(mob/user, slot)
+	. = ..()
+	if(slot != ITEM_SLOT_ICLOTHING)
+		return
+	check_physique(user)
+	add_tail_overlay(user)
+	apply_wetsuit_status_effect(user)
+	update_appearance()
+
+/obj/item/clothing/under/akula_wetsuit/dropped(mob/user)
+	. = ..()
+	if(tail_overlay)
+		user.cut_overlay(tail_overlay)
+		tail_overlay = null
+	remove_wetsuit_status_effect(user)
+	update_appearance()
+
+/// This will check the wearer's bodytype and change the wetsuit worn sprite according to if its male/female
+/obj/item/clothing/under/akula_wetsuit/proc/check_physique(mob/living/carbon/human/user)
+	icon_state = base_icon_state
+	if(user.physique == FEMALE)
+		icon_state = "[icon_state]_f"
+	return TRUE
+
+/// If the wearer has a compatible tail for the `tail_overlay` variable, render it
+/obj/item/clothing/under/akula_wetsuit/proc/add_tail_overlay(mob/living/carbon/human/user)
+	if(!user.dna.species.mutant_bodyparts["tail"])
+		return
+	var/tail = user.dna.species.mutant_bodyparts["tail"][MUTANT_INDEX_NAME]
+	switch(tail)
+		if("Akula")
+			tail_overlay = mutable_appearance('modular_skyrat/master_files/icons/mob/clothing/under/akula.dmi', "overlay_akula", -(BODY_FRONT_LAYER-0.1))
```

Comment:
Please make that .dmi file path into a define.

Also, the layer variable could probably be a define as well, while you're there.